invenio_jsonschemas/jsonresolver.py

```diff
@@ -41,3 +43,8 @@ def jsonresolver_loader(url_map):
         "{0}/<path:path>".format(current_app.config['JSONSCHEMAS_ENDPOINT']),
         endpoint=current_app.extensions['invenio-jsonschemas'].get_schema,
         host=current_app.config['JSONSCHEMAS_HOST']))
+
+
+loader = json_loader_factory(LocalProxy(lambda: jsonresolver.JSONResolver(
+    entry_point_group='invenio_records.jsonresolver'
```

Comment:
It does not work without it. If I understand correctly that entry-point enables [this hook](https://github.com/inveniosoftware/invenio-jsonschemas/blob/master/invenio_jsonschemas/jsonresolver.py#L34) that adds the url_map so that a ref like `{'$ref': 'http://localhost/schemas/hep.json'}` gets resolved correctly.